connaisseur/alert.py

```diff
@@ -168,10 +166,21 @@ def call_alerting_on_request(admission_request, *, admitted):
                 normalized_image.tag,
             )
         )
+    if images == [hook_image]:
+        return False
+    if no_alerting_configured_for_event(admitted) is True:
```

Comment:
`if True is True` ? just `return not no_alerting_configured_for_event(admitted)`